mod_ci/controllers.py

```diff
@@ -904,11 +904,11 @@ def blocked_users():
         # Define addUserForm processing
         add_user_form = AddUsersToBlacklist()
         if add_user_form.validate_on_submit():
-            if BlockedUsers.query.filter_by(userID=add_user_form.userID.data).first() is not None:
+            if BlockedUsers.query.filter_by(user_id=add_user_form.user_id.data).first() is not None:
```

Comment:
This was a careless replace, or you forgot to push your updated forms ;)

Reply:
Yeah I forgot to stage the form, I had replaced all throughout the repo.